test/lib/sstable_test_env.hh

```diff
@@ -33,10 +35,12 @@ class test_env {
         cache_tracker cache_tracker;
         test_env_sstables_manager mgr;
         reader_concurrency_semaphore semaphore;
+        compaction_manager compaction_manager;
```

Comment:
Does it need to be stopped?

This expanding the scope of test_env (from testing single sstables to testing compaction), maybe we need a different test_env for that.